generator/parseXML/ParseXML.py

```diff
@@ -779,29 +851,36 @@ def parse_deviser_xml(self):
         global_variables.add_additional_declaration(add_declarations)
 
         # get package information
+        # TODO Should these numbers be hardcoded here? Maybe better in their own file?
```

Comment:
Add an issue for this so it gets discussed.

Reply:
Now issue 17